cmd/agent/main.go

```diff
@@ -1,3 +1,51 @@
 package main
 
-func main() {}
+import (
+	"fmt"
+	send "go-yandex-metrics/cmd/agent/handlers/send"
+	"log"
+	"log/slog"
+	"strconv"
+
+	save "go-yandex-metrics/cmd/agent/handlers/save"
+
+	"go-yandex-metrics/internal/storage"
+	"os"
+	"runtime"
+	"time"
+)
+
+var logger = slog.New(slog.NewTextHandler(os.Stdout, nil))
+
+func main() {
+
+	storage.ParseFlags()
+
+	m := new(runtime.MemStats)
+	runtime.ReadMemStats(m)
+
+	pollInterval, err := strconv.ParseUint(storage.PollInterval, 10, 64)
+	if err != nil {
+		log.Fatal(err)
+	}
+	tickerSend := time.NewTicker(time.Duration(pollInterval) * time.Second)
+	fmt.Println("Poll interval is", pollInterval)
+
+	reportInterval, err := strconv.ParseUint(storage.ReportInterval, 10, 64)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Стоит пояснить, что именно произогло:


```suggestion
		log.Fatal("failed to parse %s as a report interval value: %v", storage.ReportInterval, err)
```

Reply:
Вот так видимо неправильно:

return fmt.Errorf("error parsing ReportInterval: %w", err)

?

Reply:
Сделал как вы советуете.

Reply:
Сделал:
log.Fatal(fmt.Printf("failed to parse %s as a report interval value: %v", cfg.Agent.ReportInterval, err))